### PR TITLE
Documenting SaaS Environment Management

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds a new topic about how to create and manage SaaS environments in Magento.

## Affected DevDocs pages

-  New topic

### CODE REVIEW

File: src/guides/v2.3/config-guide/saas/environment.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - Setup
+---
+
+Magento allows you to link your Magento store to a SaaS service through the Admin UI. By configuring and managing SaaS Environments, you can connect to services such as Product Recommendations.
+
+This topic describes how to create and manage your SaaS projects and environments so you can connect to these SaaS Services.
+
+## Introduction
+
+When you create a Magento account, you are entitled to one MageID. The owner of the MageID can generate a Magento API key, which is necessary to unlock the SaaS services within Magento. Once that API key is generated, the systems integrator or development team can create and manage the SaaS environments on behalf of the license holder. Each MageID can have one SaaS project and within that SaaS project you can have zero or more SaaS environments depending on the type of Magento license you have:
```
Comment (misha-kotov) [inline src/guides/v2.3/config-guide/saas/environment.md:16]: The MageID is an identifier of a Magento account, so I would not talk about it like it's an entitlement. A single user can technically create many Magento accounts with different email addresses, nothing stops that today. I rearranged some of your content and came up with the following: 

"When you create a Magento account, identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. To connect to Magento Commerce SaaS services, the API key needs to be generated by the license-holder. Then the API key can be passed to X and Y, who can then manage the SaaS projects/envs on behalf of the license-holder. 
Each MageID can have one SaaS project and within that SaaS project you can have one or more SaaS environments depending on the type of Magento license you have: "


Also add: SaaS projects/evs are separate but analogous concepts to Magento Cloud projects and environments.

File: src/guides/v2.3/config-guide/saas/environment.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+-  **Commerce** - One production environment; two testing environments
+
+-  **Open Source** - One production environment; no testing environments
+
+## Create SaaS Environment ID
```
Comment (misha-kotov) [inline src/guides/v2.3/config-guide/saas/environment.md:22]: just "SaaS Environment"

File: src/guides/v2.3/config-guide/saas/environment.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+To create a SaaS Environment ID, request the Magento API key from the Magento license holder for your store.
```
Comment (misha-kotov) [inline src/guides/v2.3/config-guide/saas/environment.md:24]: same: "SaaS Environment"

**Comment by erikmarr** [comment]
This content has been migrated over the Merchdocs repo:
https://github.com/magento/merchdocs/pull/446
Closing this PR.

**Pull request closed without merging**